examples/tests/linuxbuild.py

```diff
@@ -24,7 +24,8 @@ def setUp(self):
 
         self.linux_build = kernel.KernelBuild(kernel_version,
                                               linux_config,
-                                              self.srcdir)
+                                              self.srcdir,
+                                              ['/var/tmp'])
```

Comment:
Yes, it was here: https://github.com/avocado-framework/avocado/pull/1060
And not here: https://github.com/avocado-framework/avocado/pull/1062
But I cannot say why it disappeared :)
Anyway, agreed it's valuable.